crates/relayer/src/chain/cosmos/retry.rs

```diff
@@ -149,6 +149,16 @@ async fn do_send_tx_with_account_sequence_retry(
                     Ok(response)
                 }
 
+                Code::Err(code) if response.log.contains("packet messages are redundant") => {
+                    info!(
+                        ?response,
+                        diagnostic = ?sdk_error_from_tx_sync_error_code(code.into(), estimated_gas),
```

Comment:
Won't this just return "unknown tx error" since this error 22 is not handled in `sdk_error_from_tx_sync_error_code`, and moreover it is not an SDK error but an ibc-go one?

We can maybe just remove the diagnostic flag, and change the message to "all packets in tx have been relayed already, no fees were consumed" or something similar

Reply:
You are right, sorry I missed that.
Fixed it here 28d857bc12b9b4e0d88ee8db0bcaebbd3cecdb56
I kept the diagnostic flag, but set it to the response log and added a `code` field

Reply:
And added the additional information about the fees here c8f7bd9f481b9f10f25d0a4269a13814420de0e9